unstract/filesystem/src/unstract/filesystem/exceptions.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework.exceptions import APIException
```

Comment:
@muhammad-ali-e since you're importing from DRF here, please specify it as a dependency in this internal library

Reply:
Changed it to Exception.